code/modules/admin/watchlist.dm

```diff
@@ -36,6 +36,8 @@
 	if(browse)
 		watchlist_show(target_sql_ckey)
 
+	add_note(target_ckey, "Added to Watchlist - [reason]", null, usr.ckey, 0, null, 0)
```

Comment:
@Iamgoofball is this a snip at someone? Typically you wouldn't disclose metagame notes, and if you are being watched for consistent behavior you probably have several nonsecret notes relating to the watchlist, so it shouldn't be any surprise to someone.